portfolio/src/main/webapp/script.js

```diff
@@ -145,4 +145,93 @@ function dark() {
     document.getElementById('poem_1').style.backgroundColor = 'black';
     document.getElementById('poem_2').style.backgroundColor = 'black';
     document.getElementById('poem_3').style.backgroundColor = 'black';
-}
\ No newline at end of file
+}
+
+var amplitude = 70;
```

Comment:
Please use `let` instead of `var`.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let